bridge_ui/src/components/Transfer/PendingVAAWarning.tsx

```diff
@@ -0,0 +1,36 @@
+import { Link, makeStyles } from "@material-ui/core";
+import { Alert } from "@material-ui/lab";
+import { useSelector } from "react-redux";
+import { selectTransferSourceChain } from "../../store/selectors";
+import { CHAINS_BY_ID } from "../../utils/consts";
+
+const useStyles = makeStyles((theme) => ({
+  alert: {
+    marginTop: theme.spacing(1),
+    marginBottom: theme.spacing(1),
+  },
+}));
+
+const PendingVAAWarning = () => {
+  const classes = useStyles();
+  const sourceChain = useSelector(selectTransferSourceChain);
+  const chainName = CHAINS_BY_ID[sourceChain]?.name || "unknown";
+  const message = `The daily notional value limit for transfers on ${chainName} has been exceeded. As
+      a result, the VAA for this transfer is pending. If you have any questions,
+      please open a support ticket on `;
```

Comment:
"open a support ticket on " ... Should there be a destination?  Or is that somewhere else?

Reply:
the url is below in the <Link /> tag